libs/react-helpers/src/hooks/index.ts

```diff
@@ -6,3 +6,4 @@ export * from './use-outside-click';
 export * from './use-resize';
 export * from './use-screen-dimensions';
 export * from './use-theme-switcher';
+export * from './__generated__';
```

Comment:
shared types should be in types package no?

Reply:
Hm not sure I follow. This is just types from the use-network-params hook. react-helpers seemed like the most sensible place for this hook and the resulting generated types.